include/iocore/net/ConnectionTracker.h

```diff
@@ -71,23 +71,27 @@ class OutboundConnTrack
 
   /// Per transaction configuration values.
   struct TxnConfig {
-    int max{0};                ///< Maximum concurrent connections.
-    int min{0};                ///< Minimum keepalive connections.
-    MatchType match{MATCH_IP}; ///< Match type.
+    int client_max{0};                ///< Maximum concurrent client connections.
+    int server_max{0};                ///< Maximum concurrent server connections.
+    int server_min{0};                ///< Minimum keepalive server connections.
+    MatchType server_match{MATCH_IP}; ///< Server match type.
   };
 
   /** Static configuration values. */
   struct GlobalConfig {
-    std::chrono::seconds alert_delay{60}; ///< Alert delay in seconds.
+    std::chrono::seconds client_alert_delay{60}; ///< Alert delay in seconds.
+    std::chrono::seconds server_alert_delay{60}; ///< Alert delay in seconds.
   };
 
   // The names of the configuration values.
   // Unfortunately these are not used in RecordsConfig.cc so that must be made consistent by hand.
   // Note: These need to be @c constexpr or there are static initialization ordering risks.
-  static constexpr std::string_view CONFIG_VAR_MAX{"proxy.config.http.per_server.connection.max"};
-  static constexpr std::string_view CONFIG_VAR_MIN{"proxy.config.http.per_server.connection.min"};
-  static constexpr std::string_view CONFIG_VAR_MATCH{"proxy.config.http.per_server.connection.match"};
-  static constexpr std::string_view CONFIG_VAR_ALERT_DELAY{"proxy.config.http.per_server.connection.alert_delay"};
+  static constexpr std::string_view CONFIG_CLIENT_VAR_MAX{"proxy.config.http.per_client.connection.max"};
+  static constexpr std::string_view CONFIG_CLIENT_VAR_ALERT_DELAY{"proxy.config.http.per_client.connection.max"};
```

Comment:
"proxy.config.http.per_client.connection.alert_delay"?

Reply:
True. :) 

I should also add stats for the table size.

Reply:
Updated.